rotkehlchen/db/schema.py

```diff
@@ -797,7 +797,7 @@
 CREATE TABLE IF NOT EXISTS ens_mappings (
     address TEXT NOT NULL PRIMARY KEY,
     ens_name TEXT UNIQUE,
-    last_update TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    last_update INTEGER NOT NULL
```

Comment:
This is just to store seconds since epoch instead of the date?

Reply:
yes as we do in every other table ... huge difference. 4 bytes as opposed to 24+ bytes.